website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,287 @@
+
+## How to use this component
+
+When using this component, there are a few things you have to consider:
+
+- what toggle to use, the standard [InfoText toggle](#with-infotext-toggle) or a [generic toggle](#with-a-generic-toggle), and in that case if the generic toggle contains interactive elements or not
+- if the toggle is a [standalone element](#as-a-standalone-element) or if instead lives [inline with other text](#inline-with-other-text)
+- how the end-user should [interact](#interactivity) with the toggle to show/hide the tooltip
+
+Depending of these factors, there are different ways to implement the code. We describe these alternatives below.
+
+### With `InfoText` toggle
+
+#### As a standalone element
+
+The toggle element (and the parent "wrapping" container) are rendered by default as block elements. This means it can easily be used as standalone element.
+
+The invocation requires a "toggle" element and a "content" to be passed as yielded sub-components:
+
+```handlebars
+<Hds::RichTooltip as |RT|>
+  <RT.ToggleInfoText @icon="info">More info</RT.ToggleInfoText>
+  <RT.PopoverContent>
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+  </RT.PopoverContent>
+</Hds::RichTooltip>
+```
+
+Here we're using the `InfoText` toggle element, which ensures perceivability and accessibility out of the box. It consist of a text (with a specific underline decoration applied to it) and an optional icon (can be leading or trailing).
+
+The text and the icon are rendered inside an HTML `<button>` element (the one that technically acts as a toggling control for the popover). For details how the user can interact with this button see the [Interactivity](#interactivity) sub-section.
+
+!!! Info
+
+If this doesn't work for your needs/context, you can use a [generic toggle](#with-a-generic-toggle), but in this case it's up to you to make sure the implementation is compliant with the [accessibility requirements](/components/rich-tooltip?tab=accessibility).
+
+!!!
+
+The `PopoverContent` element instead is a pure container that yields the children inside the "popover" tooltip bubble. You can pass whatever content you need to it, but you are also responsible of styling and structuring it according to your needs/context.
+
+
+!!! Info
+
+Notice: we apply a CSS reset (`all: initial`) to the container, to avoid that styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support).
```

Comment:
```suggestion
Notice: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support).
```